src/commands/sync.ts

```diff
@@ -320,16 +324,6 @@ export default class Sync extends BaseCommand<typeof Sync> {
     // local files/folders that should never be published
     this.ignorer = new FSIgnorer(this.dir, ["node_modules", ".gadget", ".git"]);
 
-    this.watcher = new FSWatcher({
-      ignored: (filepath) => this.ignorer.ignores(filepath),
-      // don't emit an event for every watched file on boot
-      ignoreInitial: true,
-      // make sure stats are always present on add/change events
-      alwaysStat: true,
-      // wait for the entire file to be written before emitting add/change events
-      awaitWriteFinish: { pollInterval: this.flags["file-poll-interval"], stabilityThreshold: this.flags["file-stability-threshold"] },
```

Comment:
We can 🔥 these flags since we aren't using them anymore

https://github.com/gadget-inc/ggt/blob/renames/src/commands/sync.ts#L93-L108